deb/build

```diff
@@ -40,4 +40,22 @@ fpm -s dir -t deb -n "$pkgname" -v "$pkgversion" \
 	--depends libtool \
 	--deb-changelog "$changelog" \
 	--deb-no-default-config-files \
-	install/=/
+	install/usr/lib/=/usr/lib/
+
+pkgname=libdrizzle-redux-dev
```

Comment:
I don't really see the difference between this and the `pkgname=libdrizzle-redux` section that includes the header files. Am I missing something?